packages/stores/src/queries-external/store.ts

```diff
@@ -19,17 +18,23 @@ export class QueriesExternalStore extends HasMapStore<QueriesExternal> {
 export class QueriesExternal {
   public readonly queryGammPoolFeeMetrics: DeepReadonly<ObservableQueryPoolFeesMetrics>;
 
-  constructor(kvStore: KVStore) {
-    this.queryGammPoolFeeMetrics = new ObservableQueryPoolFeesMetrics(kvStore);
+  constructor(
+    kvStore: KVStore,
+    feeMetricsBaseURL = "https://api-osmosis.imperator.co"
+  ) {
```

Comment:
Probably better to declare this as a constant so that it can easily be changed or injected in Vercel as an environment variable, rather than hardcoding here?

Reply:
Yeah it's a default param, I actually prefer hardcoding vs creating 100s of env vars to keep track of since we have such a tight CI/CD pipeline

Reply:
I do like env variables for everything but mainnet vs testnet can be hardcoded for now. It makes this entire process a lot easier.